docs/DeveloperGuide.md

```diff
@@ -243,7 +254,7 @@ The following sequence diagram shows how the search operation works:
 
 </div>
 
-Step 3. The user is shown the searched list. The searched list contains all contacts that are relevant to the user's search query.
+Step 4. The user is shown the searched list. The searched list contains all contacts that are relevant to the user's search query.
```

Comment:
Just a suggestion, maybe `filtered list` might be better instead of `searched list`?